src/hooks/useAccordionAnimation.tsx

```diff
@@ -0,0 +1,106 @@
+import { useCallback, useState } from "react";
+import { StyleSheet, type LayoutChangeEvent } from "react-native";
+import {
+  useAnimatedStyle,
+  useSharedValue,
+  withSpring
+} from "react-native-reanimated";
+import { IOSpacingScale, IOSpringValues } from "../core";
+
+const accordionBodySpacing: IOSpacingScale = 16;
+
+type Params = {
+  defaultExpanded?: boolean;
```

Comment:
Why not just `expanded`? 
```suggestion
  expanded?: boolean;
```